Detectors/MUON/MID/Raw/exe/CRUBareDataChecker.h

```diff
@@ -0,0 +1,56 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file   CRUBareDataChecker.h
+/// \brief  Class to check the bare data from the CRU
+/// \author Diego Stocco <Diego.Stocco at cern.ch>
+/// \date   9 December 2019
+#ifndef O2_MID_CRUBAREDATACHECKER_H
+#define O2_MID_CRUBAREDATACHECKER_H
+
+#include <cstdint>
+#include <vector>
+#include <string>
+#include <map>
+#include <gsl/gsl>
+#include "DataFormatsMID/ROFRecord.h"
+#include "MIDRaw/LocalBoardRO.h"
+
+namespace o2
+{
+namespace mid
+{
+class CRUBareDataChecker
+{
+ public:
+  bool process(gsl::span<const LocalBoardRO> localBoards, gsl::span<const ROFRecord> rofRecords, bool resetStat = true);
+  /// Gets the number of processed events
+  unsigned int getNBCsProcessed() { return mStatistics[0]; }
```

Comment:
such getter functions should be `const` annotated systematically. Can you please go through all new interfaces in this PR and fix the constness?

Reply:
You're right: it should be ok in the new version.